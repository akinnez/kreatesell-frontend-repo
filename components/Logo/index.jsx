import React from "react";
import Image from "next/image";
import LogoImg from "public/images/logo.svg";

const Logo = () => {
  return (
    <div
      style={{
        display: "flex",
        justifyContent: "center",
        padding: "20px 0 10px 0",
        marginBottom: "10px",
      }}
    >
      <Image src={LogoImg} alt="logo" width={140} height={35} />
    </div>
  );
};

export const MobileLogo = () => {
  return (
    <div
      style={{
        display: "flex",
        justifyContent: "center",
        padding: "10px 0",
      }}
    >
      <Image src={LogoImg} alt="logo" width={120} height={30} />
    </div>
  );
};

export default Logo;
