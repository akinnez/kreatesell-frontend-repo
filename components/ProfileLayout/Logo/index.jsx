import Image from "next/image";
import LogoImg from "public/images/logo.svg";
import style from "./index.module.scss";

const Logo = () => (
  <div className={style.logoContainer}>
    <Image src={LogoImg} alt="KreateSell Logo" layout="responsive" priority />
  </div>
);

export default Logo;
