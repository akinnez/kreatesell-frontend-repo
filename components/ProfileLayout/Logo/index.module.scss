@use 'public/css/screens';

.logoContainer {
	width: 130px;
}

@include screens.minScreen('tablet') {
	.logoContainer {
		width: 170px;
	}
}
