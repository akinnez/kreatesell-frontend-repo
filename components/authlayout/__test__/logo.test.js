import Logo, { MobileLogo } from "../logo";
import renderer from "react-test-renderer";

describe("Logo :", () => {
  it("renders a snapshot of <Logo />", () => {
    const logo = renderer.create(<Logo />).toJSON();
    expect(logo).toMatchSnapshot();
  });
});

describe("MobileLogo :", () => {
  it("renders a snapshot of <MobileLogo />", () => {
    const mobileLogo = renderer.create(<MobileLogo />).toJSON();
    expect(mobileLogo).toMatchSnapshot();
  });
});
