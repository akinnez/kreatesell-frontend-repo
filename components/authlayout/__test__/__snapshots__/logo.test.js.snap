// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo : renders a snapshot of <Logo /> 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "justifyContent": "center",
      "marginBottom": "10px",
      "padding": "20px 0 10px 0",
    }
  }
>
  <div
    className="ant-image"
    style={
      Object {
        "height": undefined,
        "width": 170,
      }
    }
  >
    <img
      className="ant-image-img"
      onError={[Function]}
      onLoad={[Function]}
      src="/images/logo.svg"
      style={
        Object {
          "height": undefined,
        }
      }
    />
  </div>
</div>
`;

exports[`MobileLogo : renders a snapshot of <MobileLogo /> 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "justifyContent": "center",
      "marginBottom": "10px",
      "padding": "20px 0 10px 0.5rem",
    }
  }
>
  <div
    className="ant-image"
    style={
      Object {
        "height": undefined,
        "width": 130,
      }
    }
  >
    <img
      className="ant-image-img"
      onError={[Function]}
      onLoad={[Function]}
      src="/images/logo.svg"
      style={
        Object {
          "height": undefined,
        }
      }
    />
  </div>
</div>
`;
